### DISCUSSION TITLE
Looks like the "Developing Your Own Runtime section" link is broken

### DISCUSSION THREAD

**Discussion by behrangsa** [vercel/community#2048] [Bug Report]
### Summary

Hi,

On this page: https://vercel.com/docs/concepts/functions/serverless-functions/runtimes, the following anchor link doesn't seem to exist:

https://vercel.com/docs/concepts/functions/serverless-functions/runtimes#advanced-usage/developing-your-own-runtime

**Comment by amyegan** [comment]
Thanks for letting us know about the broken link! I'll get it fixed right away. In the meantime, you can find more info about developing your own runtime at https://vercel.com/docs/concepts/functions/serverless-functions/runtimes#developing-your-own-runtime and in the [Runtime Developer Reference](https://github.com/vercel/vercel/blob/main/DEVELOPING_A_RUNTIME.md)
